server/cmake/modules/FindLibClang.cmake

```diff
@@ -16,7 +16,7 @@
 # most recent versions come first
 # http://llvm.org/apt/
 set(LIBCLANG_KNOWN_LLVM_VERSIONS 5.0.0 5.0
```

Comment:
I'm wondering why it is found then, this does not look like a standard CMake search path.
Ok thank you, it make sense, the variable was added quite recently (and was broken until yesterday: https://reviews.llvm.org/D30911).

We should be able to rely on this however for the next version of Clang.